client/index.js

```diff
@@ -3,6 +3,7 @@
  */
 
 import {AppRegistry} from 'react-native';
+import 'react-native-gesture-handler';
```

Comment:
why need this?

Reply:
this package must be imported in order to use react-navigation, otherwise, it will break

Reply:
it should be in App.js, isn't it?

Reply:
it can be imported there as well, just personal reference here.

Reply:
we use Nav in App.js. I think the proper place is in App.js

Reply:
agreed.